src/test/java/org/stellar/sdk/ClaimableBalanceIdTest.java

```diff
@@ -0,0 +1,105 @@
+package org.stellar.sdk;
+
+import com.google.common.collect.Lists;
+import org.junit.Test;
+import org.stellar.sdk.xdr.CryptoKeyType;
+import org.stellar.sdk.xdr.MuxedAccount;
+import org.stellar.sdk.xdr.Uint64;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class ClaimableBalanceIdTest {
+
+
+  @Test
+  public void testClaimableBalanceIds() throws IOException {
+    String sourceAccount = KeyPair.fromSecretSeed("SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4").getAccountId();
+    CreateClaimableBalanceOperation op0 = new CreateClaimableBalanceOperation.Builder(
+        "420",
+        new AssetTypeNative(),
+        Lists.newArrayList(
+            new Claimant(
+                "GCACCFMIWJAHUUASSE2WC7V6VVDLYRLSJYZ3DJEXCG523FSHTNII6KOG",
+                new Predicate.Unconditional()
+            )
+        )).build();
+    Transaction transaction = new Transaction.Builder(AccountConverter.enableMuxed(), new Account(sourceAccount, 123l), Network.TESTNET)
+        .addOperation(op0)
+        .addOperation(new BumpSequenceOperation.Builder(2l).build())
+        .addOperation(op0)
+        .setTimeout(Transaction.Builder.TIMEOUT_INFINITE)
+        .setBaseFee(Transaction.MIN_BASE_FEE)
+        .build();
+
+    String expectedIdIndex0 = "0000000095001252AB3B4D16ADBFA5364CE526DFCDA03CB2258B827EDBB2E0450087BE51";
```

Comment:
Why are they uppercase? I don't think `GET /claimable_balance/:id` is case-agnostic, so we'd want to make sure they're compatible.

Reply:
Either in the encoding or in the request builder, not sure what's more appropriate